### DISCUSSION TITLE
One more idea: display recyclable commits.

### DISCUSSION THREAD

**Discussion by wl2776** [sourcegit-scm/sourcegit#832] [Ideas]
SmartGit has one more very nice feature: "Recyclable commits". 

When enabled, it shows commits that are not currently pointed to by any branch or tag.
This feature might be very useful for recovering of accidentally removed code.
Information is retrieved from `git reflog`.

Example. Gray-colored commits have been created during squashing and rebasing of a branch.
![image](https://github.com/user-attachments/assets/1e66a127-9670-48bd-8888-4ece8ba99d13)

**Comment by love-linger** [comment]
You can toggle `Enable --reflog Option`
![image](https://github.com/user-attachments/assets/50fdc5fd-4ff0-4b82-a7a5-8d511d8aa29d)
